services/mailer/src/integration-tests/smtp-transport.spec.ts

```diff
@@ -45,8 +48,16 @@ describe("SMTP transport tests", () => {
   }).timeout(15000);
 
   it("Should return 400 - bad request response", () => {
+    const { container } = GLOBAL.bootstrap;
+    const app = container.resolve("app");
+
     const body = {};
 
     return request(app).post("/api/mailer/send").send(body).expect(BAD_REQUEST);
   });
+
+  after(async () => {
+    const { container } = GLOBAL.bootstrap;
+    container.register("mailerConfig", asValue({ ...originalMailConfig }));
+  });
```

Comment:
Do we really need this?

Reply:
@eMKoded you have right we don't need it in this case. I have changed that.